### PR TITLE
docs: added Python version Badge 

### LINKED ISSUE #1052: Add a badge for Python version in README.md
## Description
README.md should have a badge for denoting the version of python needed.

## Mocks
README of [mentorship-backend](https://github.com/anitab-org/mentorship-backend/blob/develop/README.md).

Screenshot from 2020-08-29 20-56-52

## Definition of Done
- [ ] Badge should be added with appropriate version.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by atharwa-24**
# Description
Added Python Version Badge to README file,
logos to the Existing Badges in the README file
Removed extra blanc line

Fixes #1052

# Type of Change:

- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
## before

vms
## after

vms2

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by rkpattnaik780** [comment]
Remove fixes .. from commit header, it should only be in commit body.

**Comment by atharwa-24** [comment]
> 
> 
> Remove fixes .. from commit header, it should only be in commit body.

Done

**Comment by rkpattnaik780** [comment]
> > Remove fixes .. from commit header, it should only be in commit body.
> 
> Done

Hello @atharwa-24 , you are required to change the header of your commit.
Do a `git commit --amend`, edit the header and then do a `git push origin develop`.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-
+[![Build Status](https://img.shields.io/travis/anitab-org/vms/develop?logo=travis)](https://travis-ci.org/anitab-org/vms)
+[![Coverage Status](https://coveralls.io/repos/github/anitab-org/vms/badge.svg)](https://coveralls.io/github/anitab-org/vms?branch=develop) 
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com) 
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg?logo=zulip)](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms)
+[![Python Version](https://img.shields.io/badge/python-3.6-blue.svg?logo=python)](https://www.python.org/downloads/release/python-360/)
+[![License](https://img.shields.io/github/license/anitab-org/vms?logo=gnu)](https://github.com/anitab-org/vms/blob/master/LICENSE)
```
Comment (rkpattnaik780) [inline README.md:9]: No need of a license badge yet. Please stick to the issue.

Comment (atharwa-24) [reply README.md]: yes, but it looks nice!
😄

Comment (atharwa-24) [reply README.md]: I have removed it.

**Review by rkpattnaik780 — DISMISSED** [review]
LGTM! Thanks

**Comment by atharwa-24** [comment]
> 
> 
> LGTM! Thanks

Can you plz explain why the checks r getting failed
I am not able to figure it out @rkpattnaik780

**Comment by rkpattnaik780** [comment]
> > LGTM! Thanks
> 
> Can you plz explain why the checks r getting failed
> I am not able to figure it out @rkpattnaik780
> how to fig it out?

Can you post it in the zulip chat? I am not sure why markdown related change is failing the build.

**Comment by atharwa-24** [comment]
Hey @rkpattnaik780 the build has passed!!!

**Comment by rkpattnaik780** [comment]
Undo the changes of your second commit, squash and force push again. The build  will pass it seems.

**Comment by rkpattnaik780** [comment]
It shows the older build still can you try out the following steps:
`git reset HEAD^`
`git commit` (give the appropriate commit head and commit body)
`git push -f origin develop`

**Comment by atharwa-24** [comment]
added the reason for build fail in zulip chat

**Pull request closed without merging**